### ISSUE TITLE
Testing: Confirmation carousel focus outline hidden

### ISSUE THREAD

**Issue by hawkticehurst** [microsoft/vscode#309535]
Testing #309330

If you tab through the new confirmation carousel UI you'll reach a tab location where the focus outline is rendered beneath the tool call info section. The bottom corners of the carousel also clip the focus outline. Focus outline should be above everything.

<img width="664" height="157" alt="Image" src="https://github.com/user-attachments/assets/5e43a1a4-2e2d-4f53-8343-110113f117db" />

**Comment by hawkticehurst** [comment]
Oh check other themes too cause some to have even worse rendering/clipping happening

<img width="832" height="167" alt="Image" src="https://github.com/user-attachments/assets/3377a6f6-0016-4f71-8946-e09c7daf6b54" />

**Comment by rzhao271** [comment]
Marking as verified based on the verification TPI being closed.
